mobile/native/src/ln_dlc/mod.rs

```diff
@@ -390,7 +393,32 @@ pub fn run(seed_dir: String, runtime: &Runtime) -> Result<()> {
             }
         });
 
-        runtime.spawn(track_channel_status(node.clone()));
+        runtime.spawn({
+            let mut iteration_count = 0;
+
+            while !node
```

Comment:
🤔 is this really needed. It think the channel status is derived from the shadow channel in the database.

Reply:
I failed once to accept the order because I could not send a message to the coordinator to accept the offer because I was not connected :/

Reply:
But then it's not related to the `track_channel_status` but to the `sync_position_to_subchannel_state`

Reply:
You are completely right, this might have happened in a rebase. I'll fix it.